### DISCUSSION TITLE
404 page not found when using Advanced installation with Traefik

### DISCUSSION THREAD

**Discussion by sajee** [teslamate-org/teslamate#3108] [Q&A]
I'm trying to host on EC2 using the [Advanced installation with Traefik instructions](https://docs.teslamate.org/docs/guides/traefik) but I get "404 page not found".  Any suggestions for troubleshooting? Also:

1. What directory should [.htpasswd](https://docs.teslamate.org/docs/guides/traefik#htpasswd) go? Same dir as the docker-compose.yml?
2. What should TM_ENCRYPTION_KEY be? How should it be generated?

**Comment by elpanda88** [comment]
i have the exact same behaviour as you

**Comment by Ryc1x** [comment]
Not sure if this is too late but I also had the similar problem, try use the domain name you configured in the .env to access it. 
The problem might be that Traefik configuration is set up to route requests based on the Host header, and it matches to FQDN_TM only. I tried to access it through localhost and the public IP address of the machine, they both give 404. But works on the domain name I set.

**Reply by DerReparator** [reply]
Yes, directly accessing `teslamate.mydomain.xyz` (as configured in `.env`) instead of `mydomain.xyz` worked.
